src/main/scala/mesosphere/marathon/state/AppDefinition.scala

```diff
@@ -475,7 +479,15 @@ object AppDefinition {
 
   val DefaultUpgradeStrategy: UpgradeStrategy = UpgradeStrategy.empty
 
-  val DefaultLabels: Map[String, String] = Map.empty
+  object Labels {
+    val Default: Map[String, String] = Map.empty
+
+    val DcosMigrationApiPath = "DCOS_MIGRATION_API_PATH"
+    val DcosMigrationApiVersion = "DCOS_MIGRATION_API_VERSION"
+    val DcosPackageFrameworkName = "DCOS_PACKAGE_FRAMEWORK_NAME"
+    val DcosPackageIsFramework = "DCOS_PACKAGE_IS_FRAMEWORK"
+    val SingleInstanceApp = "MARATHON_SINGLE_INSTANCE_APP"
+  }
```

Comment:
I didn't split these up into a DCOS object and a Marathon object, seemed too much overhead.